modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/HistoricalSnapshotOrdinalEntries.scala

```diff
@@ -0,0 +1,79 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.data.NonEmptyList
+import cats.syntax.all._
+
+import scala.util.control.NoStackTrace
+
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.numeric.NonNegLong
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+@derive(eqv)
+case class HistoricalSnapshotOrdinalEntries(values: Option[NonEmptyList[HistoricalSnapshotOrdinalEntry]])
+
+case object InvalidRange extends NoStackTrace
```

Comment:
The rejected data may be helpful to whoever is seeing this error.

Reply:
I will need your guidance as to how to include a message in `NoStackTrace`. I don't need a stack trace, so extending `Exception` seems like overkill.

Reply:
Added message strings to each error, per our offline discussion. However, I don't think it's being logged anywhere.